### ISSUE TITLE
dash-shaka player could not fetch the SMPTE-TT images

### ISSUE THREAD

**Issue by kumarashu123** [clappr/dash-shaka-playback#62]
When the smpte based subtitle language is selected in the player during playback, player is requesting the xml file for the bitmap based subtitles and the response xml file has the list of available bitmap images . 
The  player is able to parse the available bitmap-images list from the xml file and add those list to the cue but fails to send the request to fetch the parsed images. So the bitmap based subtitles are not displayed. 

Please, help us to fix the issue with dash-shaka-playback.Thanks in advance.

**Comment by leandromoreira** [comment]
did you see any error on network or console from the browser? it might be related to cors

**Comment by kumarashu123** [comment]
There is no Network error. I am getting one error in console 
**DOMException: Failed to execute 'endOfStream' on 'MediaSource': The MediaSource's readyState is not 'open'**

Below is the url of mpd content which i am playing
'https://livesim.dashif.org/dash/vod/testpic_2s/img_subs.mpd'

**Comment by kumarashu123** [comment]
hi, Can you tell me what is the issue?

**Comment by leandromoreira** [comment]
@kumarashu123 do you have the same problem when using the https://shaka-player-demo.appspot.com/demo/#audiolang=en-US;textlang=en-US;uilang=en-US;panel=CUSTOM%20CONTENT;build=uncompiled click on Custom content e put yours

**Comment by kumarashu123** [comment]
No, I don't have any problem when i am playing using above url.
You can try this mpd **https://livesim.dashif.org/dash/vod/testpic_2s/img_subs.mpd** content to the above url.

I have cloned dash-shaka-playback and Running dev. server there i am not getting subtitles.

**Comment by kumarashu123** [comment]
Hi any updated or I will be enhancement.

**Comment by leandromoreira** [comment]
have you tried with the lastest release?

**Comment by kumarashu123** [comment]
Yes, I have tried with latest release.
Not getting subtitles.

**Comment by kumarashu123** [comment]
Can you please tell How i can used SMPTE-TT subtitles in my app.
